### ISSUE TITLE
Possibly obscure issues with keypresses

### ISSUE THREAD

**Issue by Merytsetesh** [libsdl-org/SDL#4560]
I'm having a couple of strange issues with SDL regarding keypresses and I cannot figure out why.

I'm building in VS 2019 Community, using a decently up-to-date version of SDL2, and I'm following both the SDL2 docs and LazyFoo's tutorials on how to do things, so as far as I know my code is okay. Also, I've got my code outputting SDL keysym.sym values to the console whenever I press a key, so I can check that keystrokes are recorded.

1. **Hard crash on F12**. Whenever I hit F12 I get a hard crash out. The call stack shows somewhere in nt.dll, in a worker thread. This happens only on raw F12: any other key combo seems to generate the same keysym, 1073741893.

2. **Can't generate Shift-Tab-1**. I'm using Tab plus number keys, and shift-Tab plus number keys, to generate effects in my program. I can get Shift-Tab-2 to 8 and 9 just fine, but Shift-Tab-1 doesn't seem to throw back any keypress event at all. I know these are weird combinations, but the heart has its reasons. :-D

**Comment by slouken** [comment]
I believe F12 is intercepted by the debugger as a break request. I think that will work if you run the code outside the debugger.

I have no idea why Shift-Tab-1 wouldn't work. Could it be this?
https://gaming.stackexchange.com/questions/6669/how-do-i-remove-the-limit-on-pc-keyboard-button-presses

**Comment by Merytsetesh** [comment]
Thank you for your reply. I wasn't aware that F12 is bound directly to the debugger: I tried running outside and it does indeed work.

I'll try Shift-Tab-1 with another, external, keyboard and see if it works. If not, I'll go with Ctrl-Tab-1, etc., which I've tried and does work. It would be a little frustrating as it would break the system I have, but if it's a hardware issue then it's the best way around the problem.

Thank you again.
